tests/test_group.py

```diff
@@ -0,0 +1,117 @@
+        group_info = search_listings.open_group(group_name)
+
+        # Check that everything was updated
+        assert new_group_description_text == group_info.description_text
+        assert new_group_irc_channel == group_info.irc_channel
+
+        # Delete group
+        group_info.delete_group()
+
+    @pytest.mark.credentials
+    def test_group_deletion_confirmation(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        delete_form = group.description.delete_group
+
+        assert not delete_form.is_delete_button_enabled
+
+        delete_form.check_acknowledgement()
+
+        assert delete_form.is_delete_button_enabled
+
+        groups_page = delete_form.click_delete_group()
+        assert groups_page.is_group_deletion_alert_present
+
+    @pytest.mark.credentials
+    def test_group_type_change(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        group_type = group.access.group_type
+
+        assert not group_type.is_member_criteria_visible
+
+        group_type.set_reviewed_group_type()
+
+        assert group_type.is_member_criteria_visible
+
+    @pytest.mark.credentials
+    def test_group_invitations(self, base_url, selenium, vouched_user):
+        group_name = (time.strftime('%x-%X'))
+
+        home_page = Home(base_url, selenium)
+        home_page.login(vouched_user['email'], vouched_user['password'])
+        settings = home_page.header.click_settings_menu_item()
+        groups = settings.groups.click_find_group_link()
+        create_group = groups.click_create_group_main_button()
+        create_group.create_group_name(group_name)
+        group = create_group.click_create_group_submit()
+
+        invite = group.invitations.invite
+
+        new_member = "Matt Brandt"
+
+        invite.invite_new_member(new_member)
+        invite.click_invite()
+
+        # get random index
+        invitations = group.invitations.invitations_list
+        random_profile = randrange(len(invitations.search_invitation_list))
+        assert new_member in invitations.search_invitation_list[random_profile].name
```

Comment:
I changed the comment. I could add a more descriptive one, but this is the same code we use for picking a profile on search too. https://github.com/mozilla/mozillians-tests/blob/master/tests/test_search.py#L36-L39